### ISSUE TITLE
Add kaku shortcut in toolbar

### ISSUE THREAD

**Issue by EragonJ** [EragonJ/Kaku#403]
As title, it would be cool that users can easily manipulate some simple things from the toolbar or to see the information of this song.

Worth to investigate this feature ! 

<img width="192" alt="2016-07-01 11 31 58" src="https://cloud.githubusercontent.com/assets/68322/16511144/c776f124-3f7f-11e6-9e56-658f9a5afe08.png">

**Comment by neighborhood999** [comment]
EJ, shortcut mean in `main.js`'s `shortcuts`?

```js
const shortcuts = [
  'MediaNextTrack',
  'MediaPreviousTrack',
  'MediaPlayPause',
  'Escape'
];
```

**Comment by EragonJ** [comment]
@neighborhood999 nono, this one is different thing ! it will be a small icon on the toolbar (check the demo image) and when you click it, it will show you what's the current playing song, and some handy buttons to let you go to next song / previous song ... etc.

**Comment by neighborhood999** [comment]
OK, yesterday I try to add this feature, but playing song, pause song, next, previous not yet.
![Imgur](http://i.imgur.com/YNWtk5k.png)

**Comment by EragonJ** [comment]
@neighborhood999 coool so what will happen when you click on the icon right now ?

**Comment by neighborhood999** [comment]
Only have `Help` and `Exit` two options XD, `Help` option contain `Docs` and `Issue`.

**Comment by EragonJ** [comment]
cool ! `exit = close Kaku` is super useful too  !

**Comment by EragonJ** [comment]
fixed by #406
